src/Subscriber.ts

```diff
@@ -12,29 +12,29 @@ export class Subscriber<T> extends Subscription implements Observer<T> {
   static create<T>(next?: (x?: T) => void,
                    error?: (e?: any) => void,
                    complete?: () => void): Subscriber<T> {
-    return new SafeSubscriber<T>(next, error, complete);
+    return new SafeSubscriber<T>(null, next, error, complete);
   }
 
+  public syncErrorValue: any = null;
+  public syncErrorThrown: boolean = false;
+  public syncErrorThrowable: boolean = true;
```

Comment:
@blesh they do, since they need to be accessible to Observable's [`subscribe`](https://github.com/trxcllnt/RxJS/blob/unsubscribe-on-sync-throws/src/Observable.ts#L116).